models/cs/run_bart_val.py

```diff
@@ -18,7 +18,7 @@
 import bart
```

Comment:
Update the [models/cs/README.md](https://github.com/facebookresearch/fastMRI/blob/master/models/cs/README.md) file explaining how the run commands differ for brain MRIs and knee MRIs (`--mask-type equispaced` for brains and `--mask-type random` for knees).